orchestra/migrations/0074_sanitycheck.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
won't merge this---will wait for @paopow 's PR, rebase, and rebuild migrations.

Reply:
It's fine really. :D